tests/ui/pages/desktop/home.py

```diff
@@ -39,3 +52,106 @@ def users(self):
                 """Number of users that have downloaded the extension"""
                 users_str = self.find_element(*self._users_locator).text
                 return int(users_str.split()[0].replace(',', ''))
+
+    class FeaturedExtensions(Region):
+        """ Featured Extension region """
+
+        _root_locator = (By.ID, 'featured-extensions')
+
+        _extension_locator = (By.CSS_SELECTOR, 'section > li > div')
+        _see_all_locator = (By.CSS_SELECTOR, 'h2 > a')
+
+        @property
+        def extensions(self):
+            return [self.Extension(self.page, el) for el in self.find_elements(
+                *self._extension_locator)]
+
+        class Extension(Region):
+
+            _name_locator = (By.CSS_SELECTOR, 'h3')
+            _link_locator = (By.TAG_NAME, 'a')
+
+            def see_all(self):
+                """
+                Clicks the 'See All' link within the Featured Extensions
+                region.
+                """
+                self.find_element(*self._link_locator).click()
+                from pages.desktop.details import Details
+                return Details(
+                    self.selenium, self.page.base_url).wait_for_page_to_load()
+
+            @property
+            def name(self):
+                return self.find_element(*self._name_locator).text
+
+    class FeaturedThemes(Region):
+        """
+        Featured Themes region
+        """
+
+        _root_locator = (By.ID, 'featured-themes')
+
+        _themes_locator = (By.CSS_SELECTOR, "li")
+        _see_all_link = (By.CLASS_NAME, 'seeall')
+
+        @property
+        def themes(self):
+            """
+            Represents all themes found within the Featured Themes region.
+            """
+            return [self.Theme(self, el) for el in self.find_elements(
+                *self._themes_locator)]
+
+        def see_all(self):
+            """
+            Clicks the 'See All' link within the featured Themes region.
+            """
+            self.find_element(*self._see_all_link).click()
+            from pages.desktop.themes import Themes
+            return Themes(
+                self.selenium, self.page.base_url).wait_for_page_to_load()
+
+        class Theme(Region):
+
+            _name_locator = (By.CSS_SELECTOR, "h3")
+
+            @property
+            def name(self):
+                """ Theme Name"""
+                return self.find_element(*self._name_locator).text
+
+    class FeaturedCollections(Region):
+        """ Featured Collections region """
+
+        _root_locator = (By.ID, 'featured-collections')
+        _items_locator = (By.CSS_SELECTOR, "li")
+        _see_all_link = (By.CSS_SELECTOR, "h2 a")
+
+        @property
+        def collections(self):
+            """
+            Represents all Collections found within the Featured Collections
+            """
+            return[self.Collection(self.page, el) for el in self.find_elements(
```

Comment:
whitespace after `return` and before the list-comprehension. Otherwise same style-note as in https://github.com/mozilla/addons-server/pull/5065/files#diff-df48dd32a30cf7f4b4ba891ed17bcbddR65